### ISSUE TITLE
game won't fully boot

### ISSUE THREAD

**Issue by voidwalker604** [masterfeizz/ctrQuake#23]
I am on old 3ds 11.13.0.-45 using luma3ds ver. 1.10. booted using .cia (3dsx doesn't work)
my game won't fully boot and displays
FindFile: can't find gfx/pop.tmp
Playing shareware version.
FindFile: can't find gfx.wad
Sys_error: W_loadWadFile: couldn't load gfx.wad
Press start to exit (this line displays as lines and dots until you press start)
I have tried reinstalling multiple times and this error always displays

**Comment by Wozza123** [comment]
Yes it's exactly the same for me. The instructions you've posted do not work at all. The [Quake2CTR](https://github.com/masterfeizz/Quake2CTR/releases/tag/v0.1) cia works fine but not the ctrQuake cia.

I tried unzipping to the SD card's root and also to my 3ds folder but neither worked. Running from Homebrew also fails.

Also, inside your ctrQuake-master zip is a duplicate of the ctrQuake-master folder. So it's in there twice kinda thing.

Please update your instructions and files.

**Comment by frans4080** [comment]
same thing here i tried playing this on an emulator and it gave the same error message.
